optimum/bettertransformer/transformation.py

```diff
@@ -246,7 +246,6 @@ def transform(
             remove_hook_from_module(model, recurse=True)
 
         if keep_original_model:
-            model = model.requires_grad_(False)
```

Comment:
is this fine to remove this for encoder models?

Reply:
actually no, let me propose a fix